lib/backend/providers.dart

```diff
@@ -42,7 +45,15 @@ class _ProvidersSingleton {
 
   Future<void> _init() async {
     if (_inited) return;
-    _isar = await openIsar();
+    WidgetsFlutterBinding.ensureInitialized();
+    final dir = await getApplicationDocumentsDirectory();
+    _isar = await Isar.open(
+      schemas: [EntityStatSchema],
+      directory: p.join(
+        dir.path,
+        'isar',
```

Comment:
nit: removing this trailiing comma imo would look better

Reply:
done. I see that was unnecessary as line length was < 80